CHANGELOG.md

```diff
@@ -1,85 +1,119 @@
 # Changelog
 
+## 0.8.5
+
+- AgentLogger now always needs a filename. However, the MAS utility variable_logging
```

Comment:
Maybe explain in the issue why that is the case. I dont get it. And reference #44 here